File: Android/Les4ElefantCowork/app/src/main/java/com/les4elefantastiq/les4elefantcowork/models/Position.java
package com.les4elefantastiq.les4elefantcowork.models;

/**
 * Created by Math on 06/08/16.
 */

public class Position {

    int id;
    String companyName;
    Boolean isCurrent;
    String startDate;
    String endDate;
    String title;

    public Position(int id, String companyName, Boolean isCurrent, String startDate, String endDate, String title) {
        this.id = id;
        this.companyName = companyName;
        this.isCurrent = isCurrent;
        this.startDate = startDate;
        this.endDate = endDate;
        this.title = title;
    }
}

File: Backend/CoWorkers.Domain/Position.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Comeeting.Domain
{
    public class Position
    {
        public int Id { get; set; }
        public string CompanyName { get; set; }
        public bool IsCurrent { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public string Title { get; set; }

        public string CoworkerLinkedInId { get; set; }

        public Coworker Coworker { get; set; }
    }
}
